File: Solution/1920.基于排列构建数组/1920.基于排列构建数组.cpp
/*
 * @Descroption: LeetCode 1920. 基于排列构建数组
 * @Author: EmoryHuang
 * @Date: 2021-11-05 20:21:40
 * @Method:
 * 简单题
 */

class Solution {
   public:
    vector<int> buildArray(vector<int>& nums) {
        vector<int> ans;
        for (int num : nums) ans.push_back(nums[num]);
        return ans;
    }
};

File: Solution/1920.基于排列构建数组/1920.基于排列构建数组.py
'''
Descroption: LeetCode 1920. 基于排列构建数组
Author: EmoryHuang
Date: 2021-11-05 20:20:30
Method:
简单题
'''


class Solution:
    def buildArray(self, nums: List[int]) -> List[int]:
        return [nums[num] for num in nums]